src/ServicePulse.Host/app/js/services/services.monitoring.js

```diff
@@ -29,7 +42,24 @@
             mappedUrls.forEach(function (url) {
                 $http.get(url)
                     .then(function (result) {
+                        if (connectionToasts[url] !== undefined) {
+                            var message = connectionToasts[url];
+                            // doesn't exist
+                            toastr.clear(message);
+                            connectionToasts[url] = undefined;
+                        }
                         observer.onNext(result.data);
+                    }, function (error) {
```

Comment:
I'm not entirely sure this is in the right place. Shouldn't the service merely tell the observers that the error occurred and let the observers decide whether to show a notification or not?